src/annotations/api/index.ts

```diff
@@ -55,10 +55,11 @@ export const writeAnnotation = async (
 }
 
 export const getAnnotations = async (
+  orgID: string,
   stream?: string
 ): Promise<AnnotationResponse[]> => {
-  const params: GetAnnotationsParams = {
-    query: {AnnotationListFilter: formatAnnotationQueryString({stream})},
+  const params = {
```

Comment:
Had to make this not have a specific type, otherwise it wouldn't work for the cloud annotations API which doesn't have the `orgID` parameter.